logos/src/logos/capacity/capacity_planner.py

```
@@ -829,6 +1288,77 @@ def _effective_demand(
+                continue
+            to_retry.append(model_name)
+
+        for m in stale:
+            self._pending_capacity.pop(m, None)
+
+        for m in to_retry:
+            self._pending_capacity.pop(m, None)
+            logger.info("Retrying pending capacity for %s on provider %d", m, provider_id)
+            asyncio.create_task(
+                self.prepare_lane_for_request(provider_id, m, timeout_seconds=30.0)
+            )
```

Ruff (0.15.9)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)